host_core/lib/host_core/control_interface/server.ex

```diff
@@ -202,6 +202,21 @@ defmodule HostCore.ControlInterface.Server do
     {:reply, response}
   end
 
+  # Stop Host
+  defp handle_request({"cmd", _host_id, "stop"}, body, _reply_to) do
+    # TODO: Right now this will contain a parameter for timeout. Obviously how this works currently
+    # only results in the graceful shutdowns built into the system. There may be some inflight work
+    # we want to wait for up to the timeout. We could use this library possibly so we can put in
+    # hooks: https://github.com/botsquad/graceful_stop.
+    _stop_host_command = Jason.decode!(body)
```

Comment:
This will crash the control interface server if the JSON decoding fails, thereby orphaning this host and making it unable to handle subsequent requests (like shutdowns). We should probably do something like `case Jason.decode(body) do` here, and `Logger.error` if the decoding fails, and then continue with the shutdown safely thereafter.

Reply:
Added #265 to go back and fix this